src/Controller/Organization/MissionType/MissionTypeDeleteController.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Controller\Organization\MissionType;
+
+use App\Entity\MissionType;
+use App\Entity\Organization;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\Security\Core\Exception\AccessDeniedException;
+
+/**
+ * @Route("/delete/{id}", name="app_organization_mission_type_delete", methods={"GET"})
+ */
+class MissionTypeDeleteController extends AbstractController
+{
+    public function __invoke(EntityManagerInterface $entityManager, MissionType $missionType): RedirectResponse
+    {
+        $organization = $this->getUser();
+        if (!$organization instanceof Organization || false === $organization->isParent()) {
+            throw new AccessDeniedException();
+        }
+
+        $entityManager->beginTransaction();
```

Comment:
this is already done in the flush()

Reply:
Indeed, I took it from vehicles controller but was not sure it is really usefull...

Reply:
@mRoca  what do you think about the 
```php 
$entityManager->commit();
```
statement in the same code block ? I'm wondering if it is really usefull too...

Reply:
I have my answer. commit() is not needed anymore as we removed the openTransaction() call